Analysis/output/country_data_report.Rmd

```diff
@@ -28,6 +28,7 @@ library(taxdat)
 library(sf)
 library(raster)
 library(stars)
+sf::sf_use_s2(FALSE)
```

Comment:
that's because i met some shapefile (geometry) issues for some countries and I added this to solve the problem.  but i forgot the specific issues, sorry.

Reply:
I think go ahead and remove. We should troubleshoot the individual issue directly. s2 is a bit more principled than sf, but that is helpful sometimes